src/Apps/Settings/Routes/SavedSearchAlerts/Components/SavedSearchAlertEditForm.tsx

```diff
@@ -170,8 +170,10 @@ const SavedSearchAlertEditForm: React.FC<SavedSearchAlertEditFormProps> = ({
 
               <Box>
                 <Text variant="xs">Filters</Text>
+
                 <Spacer y={2} />
-                <Flex flexWrap="wrap" mx={-0.5}>
```

Comment:
The `ActiveFilterPills` component suffered from having external margins, then all the call sites had to negate them using negative margin.

Just use `gap`.